TabNavigator.js

```diff
@@ -63,11 +63,8 @@ export default class TabNavigator extends React.Component {
         <SceneContainer key={sceneKey} selected={selected}>
           {item}
         </SceneContainer>;
-      if (selected) {
-        scenes.push(scene);
-      } else {
-        scenes.unshift(scene);
-      }
+
+      scenes.push(scene);
```

Comment:
I'd like to understand this better since reordering four keyed React components should be cheap.